#22 Adding API for GET /allusers?limit=<int>

syedautherabbas opened the pull request.

syedautherabbas committed:
#22 Adding API for GET /allusers?limit=<int>

what-name commented:
The code looks good to me. Can you please upload a screenshot of the output of the `sam/api/tests/test_get_users.py` file?

what-name committed:
Change API URIs

- `/user` -> `get_user_by_id?userid=`
- `/users` -> `get_users?limit=`
_(pseudocode)_

what-name committed:
Change APIEndpoints output export name

what-name committed:
Add -s flag to pytest to output print statements

what-name committed:
Change PyTest outputs - print lambda response

what-name committed:
Assert int os.environ limit
